CONTRIBUTING.md

```diff
@@ -1,18 +1,26 @@
-# Contributing
+# Contributing guidelines
 
 ## Topics
 * [Code contribution guidelines](#code-contribution-guidelines)
+  * [Dev Setup and debugging help](#dev-setup-and-debugging-help)
+  * [Generate Pull Requests](#pull-requests)
+  * [Merge approval process](#merge-approvals)
+* [Developer workflow](#typical-developer-workflow)
+  * [How to build?](#build-test-rinse-repeat)
+  * [Setting up local testbed](#local-testbed-setup)
+  * [Capturing ESX Code Coverage](#capturing-esx-code-coverage)
 * [Bug filing guidelines](#bug-filing-guidelines)
-* [Testing and CI](#testing-and-ci)
+* [CI/CD system](#cicd-system)
+* [Cutting a new release guidelines](#cutting-a-new-release-guidelines)
 
 ## Code Contribution guidelines
 ### Dev Setup and debugging help
 Read the [FAQ on the Wiki](https://github.com/vmware/docker-volume-vsphere/wiki#faq)
 ### Pull Requests
 * Create a fork or branch (if you can) and make your changes
-   * Branch should be suffixed with github user id: (branch name).(github user id) Example: mydevbranch.kerneltime
-   * If you want to trigger CI test runs for pushing into a branch prefix the branch with runci/ Example: runci/mylatestchange.kerneltime
-   * If you want to skip running CI test e.g. *Documentation change/Inline comment change* or when the change is in `WIP` or `PREVIEW` phase, add `[CI SKIP]` or `[SKIP CI]` to the PR title.
+   * Branch should be suffixed with github user id: `(branch name).(github user id)` Example: `mydevbranch.kerneltime`
+   * If you want to trigger CI test runs for pushing into a branch prefix the branch with `runci/` Example: `runci/mylatestchange.kerneltime`
+   * If you want to skip running CI test e.g. **Documentation change/Inline comment change** or when the change is in `WIP` or `PREVIEW` phase, add `[CI SKIP]` or `[SKIP CI]` to the PR title.
 * Each PR must be accompanied with unit/integration tests
```

Comment:
Nit: Not your change, but I think we should say "Each PR **should** be accompanied with unit/integration tests whenever possible"